src/python/series.rs

```diff
@@ -312,3 +321,45 @@ impl From<PySeries> for series::Series {
         item.series
     }
 }
+
+fn infer_daft_dtype_for_sequence(
+    vec_pyobj: &[PyObject],
+    py: pyo3::Python,
+) -> PyResult<Option<DataType>> {
+    let py_pil_image_type = py
+        .import("PIL.Image")
+        .and_then(|m| m.getattr(pyo3::intern!(py, "Image")))?;
```

Comment:
Would this code fail gracefully if PIL isn't installed in the venv?

Reply:
Ah right, I had that as a TODO but forgot to add that!

Reply:
Fixed!